registration/tests/browser/browser_registration.py

```diff
@@ -60,18 +61,47 @@ def url(self):
             path=reverse('registration:register'),
         )
 
+    @property
+    def form(self):
+        """
+        The form on the page.
+        """
+        return self.client.find_element_by_tag_name('form')
+
+    def fill_form(self, form_data):
+        """
+        Fill in the form with the given data.
+        """
+        for field, value in form_data.items():
+            try:
+                element = self.form.find_element_by_name(field)
+            except NoSuchElementException:
+                # Some form fields, like the password fields, only appear if
+                # the user is not already logged in.
+                continue
```

Comment:
@omarkhan feels like a bad idea - shouldn't caller not pass field value in field_data in such case?